tools/azure-sdk-tools/packaging_tools/sdk_generator.py

```diff
@@ -273,23 +273,22 @@ def main(generate_input, generate_output):
     result = {}
     python_tag = data.get("python_tag")
     package_total = set()
-    spec_word = "readmeMd"
+    readme_files = []
```

Comment:
Seems this array is used to keep both readme files and typespec folder paths. Can we rename it appropriately?

Reply:
Make sense and rename it to `readme_and_tsp`